### ISSUE TITLE
create "real" website (e.g. jbake)

### ISSUE THREAD

**Issue by gernotstarke** [aim42/aim42#132]
currently the aim42.github.io is just a huge pile of text - without any typical website navigation.

It'd be nice to have subpages for community, issues etc., like the following examples:

* http://jbake.org/
* http://new.griffon-framework.org/

these are great examples of site+documentation, looks way better than our "docu-only"

**Comment by mgfeller** [comment]
Good idea. Another static site generation tool is [Awestruct](http://awestruct.org/), it integrates with Asciidoctor.

**Comment by gernotstarke** [comment]
One idea: Create statically generated "portal" site (e.g. with jekyll + github-pages), and let this portal include all the aim42 practices in some kind of sub-navigation.
